internal/controller/telemetry/collector_controller.go

```diff
@@ -168,7 +168,7 @@ func (r *CollectorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			UpgradeStrategy: "none",
 			Config:          otelConfig,
 			Mode:            otelv1alpha1.ModeDaemonSet,
-			Image:           "ghcr.io/axoflow/axoflow-otel-collector/axoflow-otel-collector:0.98.0-bytes",
+			Image:           "ghcr.io/axoflow/axoflow-otel-collector/axoflow-otel-collector:0.98.0-count3",
```

Comment:
We should instead use a monotonically incremented build number here I think (later on)